client/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Booking Request</title>
+  <!-- <link rel="shortcut icon" href=""> -->
+  <!-- <link rel="stylesheet" href="styles.css"> -->
+</head>
+<body>
+  <h1>Booking Request</h1>
+<div id="app"></div>
```

Comment:
You'll probably want to change the id here to be something more detailed like "booking request". That way, in our proxy server, we can have an overall app and specific sections for all our services.

Reply:
Changed the id to 'booking' !